src/main/java/com/iexec/worker/config/BlockchainAdapterConfigurationService.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.config;
+
+import com.iexec.common.config.PublicChainConfig;
+import com.iexec.worker.feign.CustomBlockchainAdapterClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+
+import static org.web3j.protocol.core.JsonRpc2_0Web3j.DEFAULT_BLOCK_TIME;
+
+@Slf4j
+@Service
+public class BlockchainAdapterConfigurationService {
+    private final PublicChainConfig publicChainConfig;
+
+    public BlockchainAdapterConfigurationService(CustomBlockchainAdapterClient customBlockchainAdapterClient) {
+        this.publicChainConfig = customBlockchainAdapterClient.getPublicChainConfig();
+        log.info("Received public chain config [{}]", this.publicChainConfig);
```

Comment:
Our standard is `.. [blabla:{}]", blabla);`

Reply:
Yes, but we have some examples of my way [here](https://github.com/iExecBlockchainComputing/iexec-worker/blob/c13cc940ad069b33f8c8500bf8693e3ef50b8ae5/src/main/java/com/iexec/worker/tee/scone/SconeConfiguration.java#L85) and [here](https://github.com/iExecBlockchainComputing/iexec-worker/blob/c13cc940ad069b33f8c8500bf8693e3ef50b8ae5/src/main/java/com/iexec/worker/compute/TeeWorkflowConfiguration.java#L78). I find it more appropriate for our use case.
Would you prefer something like `Received data [publicChainConfig:{}]`? I could change both other occurrences such as `Received data [teeWorkflowConfiguration:{}]` and `Received data [casUrl:{}]`.

Reply:
Both
`Received public chain configuration [publicChainConfig:{}], publicChainConfig);`
and
`Received public chain configuration [config:{}], publicChainConfig); `
or even
`Received public chain configuration [time:{}ms], publicChainConfig); `
are fine to me.

You can update other logs if you find some yes :)

Reply:
I was afraid of creating duplicates, making the logs harder to read. In case of single values, it looked OK to put he value labels directly into the message instead of into the square brackets.
But if you prefer to stay consistent whether we have one or multiple values, let's do that! Fixed for all 3 occurrences :)